modules/test-runner/src/transfer/asyncTransferOffline.test.ts

```diff
@@ -132,7 +145,7 @@ describe("Async transfer offline tests", () => {
    *
    * Recipient should be able to claim payment regardless.
    */
-  it("sender successfully installs transfer, goes offline before sending paymentId/preimage, and stays offline", async () => {
+  it.skip("sender successfully installs transfer, goes offline before sending paymentId/preimage, and stays offline", async () => {
```

Comment:
is there a reason this is skipped or was it just causing problems?

Reply:
This is one that was causing issues that I wanted help with. We should probably unskip it and fix it.

Reply:
Done, now we have to deal with it.